client/README.md

```diff
@@ -0,0 +1,118 @@
+# Rect Native client for Availability
+
+This is a React Native app that consumes the GraphQL API provided by the code in
+`server/` built to support iOS and Android. To get started you should probably
+read the [React Native website](https://facebook.github.io/react-native/).
+
+The app was originally built using the wonderful
+[Chatty](https://github.com/srtucker22/chatty)
+[tutorial](https://medium.com/react-native-training/building-chatty-a-whatsapp-clone-with-react-native-and-apollo-part-1-setup-68a02f7e11).
+
+It uses:
+* The [GraphQL](http://graphql.org) protocol to talk to the server
+* [Apollo Client](https://github.com/apollographql/apollo-client) to manage
+  interactions with GraphQL
+* [Redux](https://redux.js.org) to store state on the client
+* [Geolocation](https://facebook.github.io/react-native/docs/geolocation.html)
+  as built into React Native
+
+
+## Install
+
+To get started you'll need to install [NodeJS](https://nodejs.org/en/) with
+your favourite package manager. Then install yarn:
+```sh
+npm install -g yarn
+```
+
+Install the react-native CLI tool:
+```sh
+npm install -g react-native-cli
+```
+
+Now clone the repo and cd to it:
+```sh
+git clone ...
+cd availability-poc
+```
+
+Install dependencies:
+```sh
+yarn
```

Comment:
dont you have to drop into the server folder before running yarn?

Reply:
nice find